Lib/fontmake/__main__.py

```diff
@@ -31,6 +31,9 @@ def main():
     parser.add_argument('--family-name', help='Family name to use for masters,'
                         'and to filter output instances by')
     parser.add_argument('--use-afdko', action='store_true')
+    parser.add_argument('--keep-direction', dest="reverse_direction",
+                        action='store_false', help='Do not reverse contour '
+                        'direction')
```

Comment:
I think we need a way to skip cu2qu altogether when one provides input UFOs which already contain quadratic curves. There's no point in passing through cu2qu if one already knows the input sources are already in quadratic. We were thinking about fontmake looping through the glyphs and breaking as soon as it finds a 'curve' segment if the output format is ttf, or a 'qcurve' segment if the output is otf (though converting 'qcurve' to 'curve' is not supported yet).